roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
+  with_items: "{{ __warn_deprecated_vars }}"
```

Comment:
I like the idea of a pause if any are flagged.
We have not yet refactored out multiple includes... I've noticed that with two nodes I see double the checks. +1 to singleton-izing